app/src/main/java/com/example/hcm_102_0006/android_project_m/view/adapter/AdapterGenres.java

```diff
@@ -0,0 +1,73 @@
+package com.example.hcm_102_0006.android_project_m.view.adapter;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.databinding.ObservableField;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.example.hcm_102_0006.android_project_m.R;
+import com.example.hcm_102_0006.android_project_m.service.model.Genres;
+import com.example.hcm_102_0006.android_project_m.databinding.ItemGenres2Binding;
+
+import java.util.List;
+
+/**
+ * Created by hcm-102-0006 on 22/11/2017.
+ */
+
+public class AdapterGenres extends RecyclerView.Adapter<AdapterGenres.MyViewHolder> {
+
+    private Activity mContext;
+    public static final String KEY_RESULT = "GENRES";
```

Comment:
2.8 Class member ordering

This is not the only correct solution, but it is recommended to use the following order :

Constants
Fields
Constructors
Override methods and callbacks (public or private)
Public methods
Private methods
Inner classes or interfaces